test/specs/Hw5/HotelsPOM.js

```diff
@@ -0,0 +1,138 @@
+const { expect, assert } = require("chai");
+
+const moment = require("moment");
+
+const commands = require("./commands");
+
+class homework_5_hotels {
+  
+    now = new moment();
+  
+    lCommands = new commands();
+  
+    //hotels
+  dateForm = "#date_form_field-btn";
+  goingTo = "//button[@aria-label='Going to']";
+  emptyGoingTo = "#destination_form_field";
+  doneButton = '//button[@data-stid = ("apply-date-picker")]';
+  searchButtonHotels = '//button[@id="submit_button"]';
+  locationTitle = '//div[@data-test-id = ("content-location-title")]';
+  dateTitle = '//span[@data-test-id = ("playback-date-range")]';
+
+  async dateFormClick() {
+    let click = await this.lCommands.findWebElement(this.dateForm);
+    return click.click();
+  }
+  async goingToClick() {
+    let click = await this.lCommands.findWebElement(this.goingTo);
+    return click.click();
+  }
+  async emptyGoingToText(text) {
+    let destination = await this.lCommands.findWebElement(this.emptyGoingTo);
+    return destination.setValue(text);
```

Comment:
This one also not a return function.
Again use typeInWebElement(locator, data) here from Commands page. Same observation for other functions of this page.